build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -186,16 +214,31 @@ Future _createDevCompilerModule(
   // TODO(jakemac53): Fix the ddc worker mode so it always sends back a bad
   // status code if something failed. Today we just make sure there is an output
   // JS file to verify it was successful.
+  var message = response.output.replaceAll('${scratchSpace.tempDir.path}/', '');
   if (response.exitCode != EXIT_CODE_OK || !jsOutputFile.existsSync()) {
-    var message =
-        response.output.replaceAll('${scratchSpace.tempDir.path}/', '');
     throw DartDevcCompilationException(jsId, '$message}');
   } else {
+    if (message.isNotEmpty) {
+      log.info(message);
```

Comment:
Will this be noisy? Also you can probably fit this on one line.

Reply:
ddc does not output anything under normal circumstances